### ISSUE TITLE
qs: add option to disable printing of strings with given tag

### ISSUE THREAD

**Issue by williballenthin** [mandiant/flare-floss#795]
via #761 and @r0ny123

add an option to disable the printing of common/library strings to the console.
and expanding on this: any tag specified by the user.

currently #code is hidden by default, maybe provide a way to un-hide that.

**Comment by williballenthin** [comment]
it feels like this sort of interaction is best done live, such as with a quick filter panel and re-rendering the output. this way the user can play around with the output until it works for them. i'm a little worried that relying on CLI arguments is tedious to type and format. still, we should definitely have the functionality.

**Comment by mr-tz** [comment]
I agree that this would be best done interactively. Are we in good enough shape yet to start on the dynamic UI?

**Comment by williballenthin** [comment]
i would enjoy working on the interactive aspect of QS. my only hesitation right now is that it would distract from any other effort currently in progress with QS. what do we have left to explore and do we want to figure that out now?

for example, the databases need some work, especially #expert, so i think a database editor is useful, so had been working on that. this way, QS with static output is useful standalone, before the interactive view.

**Comment by mr-tz** [comment]
ok, let's not get ahead of ourselves 😄 
maybe can just start collection (user-specific) configuration values like below for now
```json
{
  "min_str_len": 6,
  "tags_to_hide": ["#code"],
  "tags_to_show": [],
  ...
}
```

**Comment by r0ny123** [comment]
I like @mr-tz's idea of configuration values.

**Comment by mr-tz** [comment]
I think this feature should be present in the interactive UI not the CLI tool (for now)
